src/config/dynamic-theme-fixes.config

```
@@ -20112,6 +20112,11 @@ INVERT
 .c-tab-bar__logo
 .c-footer__logo-link
 
+CSS
+.\[\&_a\]\:shadow-underline-black a {
```

`.\[\&_a\]\:shadow-underline-black a` doesn't look like correct CSS to me...

```suggestion
:root {
```